client/coral-embed-stream/src/containers/Embed.js

```diff
@@ -20,12 +22,72 @@ const {logout, checkLogin} = authActions;
 const {fetchAssetSuccess} = assetActions;
 
 class EmbedContainer extends React.Component {
+  subscriptions = [];
+
+  subscribeToUpdates(props = this.props) {
+    if (props.auth.loggedIn) {
+      let sub = props.data.subscribeToMore({
+        document: USER_BANNED_SUBSCRIPTION,
+        variables: {
+          user_id: props.auth.user.id,
+        },
+        updateQuery: () => {
+          addNotification('info', t('your_account_has_been_banned'));
+        },
+      });
+
+      this.subscriptions.push(sub);
+
+      sub = props.data.subscribeToMore({
+        document: USER_SUSPENDED_SUBSCRIPTION,
+        variables: {
+          user_id: props.auth.user.id,
+        },
+        updateQuery: () => {
+          addNotification('info', t('your_account_has_been_suspended'));
+        },
+      });
+
+      this.subscriptions.push(sub);
+
+      sub = props.data.subscribeToMore({
+        document: USERNAME_REJECTED_SUBSCRIPTION,
+        variables: {
+          user_id: props.auth.user.id,
+        },
+        updateQuery: () => {
+          addNotification('info', t('your_username_has_been_rejected'));
+        },
+      });
+
+      this.subscriptions.push(sub);
+    }
+  }
```

Comment:
Since these subscriptions are so similar, I feel like there are cleaner ways to build `this.subscriptions` array. Instead of imperatively setting a `sub` repeatedly and `this.subscriptions.push(sub)`, maybe try declarative:

```
subscribeToUpdates(props = this.props) {
  if (props.auth.loggedIn) {
    const newSubscriptions = [{
      document: USER_BANNED_SUBSCRIPTION,
      updateQuery() {
        addNotification('info', t('your_account_has_been_banned'));
      },
    },
    {
      document: USER_SUSPENDED_SUBSCRIPTION,
      updateQuery() {
        addNotification('info', t('your_account_has_been_suspended'));
      },
    },
    {
      document: USERNAME_REJECTED_SUBSCRIPTION,
      updateQuery() {
        addNotification('info', t('your_username_has_been_rejected'));
      },
    }];
    
    this.subscriptions = newSubscriptions.map(s => props.data.subscribeToMore({
      document: s.document,
      variables: {
        user_id: props.auth.user.id,
      },
      updateQuery: s.updateQuery,
    }));
  }
}
```

or if you're really WILD 🔥 💯:

```
subscribeToUpdates(props = this.props) {
  if (props.auth.loggedIn) {
    const newSubscriptions = [{
      document: USER_BANNED_SUBSCRIPTION,
      t: t('your_account_has_been_banned'),
    },
    {
      document: USER_SUSPENDED_SUBSCRIPTION,
      t: t('your_account_has_been_suspended'),
    },
    {
      document: USERNAME_REJECTED_SUBSCRIPTION,
      t: t('your_username_has_been_rejected'),
    }];
		
    this.subscriptions = newSubscriptions.map(s => props.data.subscribeToMore({
      document: s.document,
      variables: {
        user_id: props.auth.user.id,
      },
      updateQuery() {
        addNotification('info', s.t);
      },
    }));
  }
}
```

This is easier for me to read and reason about at least, fewer lines, and probably less work to maintain. Your logic good though, so up to you. 👍